Default.sublime-commands

```diff
@@ -6,5 +6,14 @@
 	{
 		"caption": "Trailing Spaces: Delete Trailing Spaces",
 		"command": "delete_trailing_spaces"
-	}
+	},
+	{
+		"caption": "Preferences: Trailing Spaces Settings",
+		"command": "edit_settings",
+		"args": {
+			"base_file": "${packages}/TrailingSpaces/trailing_spaces.sublime-settings",
+			"default": "// Settings in here override those in \"TrailingSpaces/trailing_spaces.sublime-settings\",\n\n{\n\t$0\n}\n",
```

Comment:
Weird comma after the comment here.

Reply:
Removed those from both menu and commands